service/src/storage.rs

```diff
@@ -0,0 +1,74 @@
+use mysql_async::prelude::*;
```

Comment:
I think we should use sqlx to support different storage backends, and the sqlx also provides a friendly way to track database changes.  https://docs.rs/sqlx/latest/sqlx/macro.migrate.html

Reply:
ok